src/org/zaproxy/zap/extension/pscanrulesAlpha/ServerHeaderInfoLeakScanner.java

```diff
@@ -99,19 +98,25 @@ public int getPluginId() {
 	
 	@Override
 	public String getName(){
-		return Constant.messages.getString(MESSAGE_PREFIX + "name");
+		return Constant.messages.getString("pscanalpha.serverheader.scanner.name");

```

Comment:
I guess I was thinking of the config file: https://github.com/zaproxy/zaproxy/wiki/ZAP-API-Scan#configuration-file but it seems it actually contains both the ID and name, so if the names mismatch it should be irrelevant.